### DISCUSSION TITLE
Download model_name and vocoder_name without having to run tts

### DISCUSSION THREAD

**Discussion by howardbaik** [coqui-ai/TTS#2609] [General Q&A]
I am currently using the TTS Command Line Interface (CLI) to run a TTS model: 

```
tts --text "Text for TTS" \
    --model_name "<type>/<language>/<dataset>/<model_name>" \
    --out_path folder/to/save/output.wav
```

I see that when I use a new model that is NOT downloaded, the `tts` command first downloads the `model_name` and `vocoder_name` before converting text-to-speech. I was wondering if there was any built-in support downloading the desired model_name + vocoder_name beforehand. 

For ex, I would run a terminal command to download a desired model, and then run `tts` to convert text-to-speech.

**Comment by thorstenMueller** [comment]
I'm not aware of a command line argument for just downloading the tts/vocoder models. But the download urls can be taken from this file and you might download it using your own logic before calling tts.
https://github.com/coqui-ai/TTS/blob/dev/TTS/.models.json
